src/main/java/org/broadinstitute/barclay/help/DocWorkUnit.java

```diff
@@ -169,6 +172,11 @@ public void processDoc(final List<Map<String, String>> featureMaps, final List<M
      */
     public boolean getBetaFeature() { return betaFeature != null; }
 
+    /**
+     * @return a boolean determining if this documented feature is marked as an experimental feature
+     */
+    public boolean getExperimentalFeature() { return experimentalFeature != null; }
```

Comment:
Shouldn't this be (and the previous) `isExperimentalFeature()`?

Reply:
that would be a more idiomatic name

Reply:
Done, both changed.